.github/workflows/builds-review.yaml

```diff
@@ -0,0 +1,62 @@
+name: Review builds
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  build:
+    if: |
+      github.event_name == 'pull_request' &&
+      github.event.label.name == 'build::review'
+    strategy:
+      matrix:
+        include:
+          - runner: ubuntu-latest
+            subdir: linux-64
+          - runner: macos-latest
+            subdir: osx-64
+          - runner: windows-latest
+            subdir: win-64
+    runs-on: ${{ matrix.runner }}
+    steps:
+
+      - name: Remove build label
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.CANARY_ACTION_TOKEN }}
```

Comment:
@jezdez which kind of token do we need here? Just a PAT for this repo?

Reply:
I'll add one for @conda-bot

Reply:
Done